includes/discovery/sensors/temperature/timos.inc.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+if ($device['os'] === 'timos') {
+    $oids = snmp_walk($device, 'tmnxHwID', '-Osqn', 'TIMETRA-CHASSIS-MIB', 'aos');
```

Comment:
I think I got it, after a little tinkering and poking around at some of the other includes.  It does result in a much cleaner, simpler bit of code.  👍